apps/aecore/lib/aecore/persistence/worker.ex

```diff
@@ -187,6 +190,38 @@ defmodule Aecore.Persistence.Worker do
     {:reply, all_blocks, state}
   end
 
+  def handle_call(:delete_all_blocks, _from,
+    %{blocks_family: blocks_family} = state) do
+    for {key, _} <- Rox.stream(blocks_family), do: Rox.delete(blocks_family, key)
```

Comment:
Use Enum.each instead

Reply:
Why should I use Enum.each instead, is there a benefit to it? Give reason behind your suggestion

Reply:
Because for returns a list of the results for each iteration and each returns an :ok

Reply:
I don't think this would matter in this case, having in mind that we don't use the result of the for/Enum.each result. We just delete each element

Reply:
Yes, we don't use the result, that's why each should be used, using for will store each result which is an unneeded operation